DllImportGenerator/TestAssets/NativeExports/Error.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace NativeExports
+{
+    public static unsafe class Error
+    {
+        private class Kernel32
+        {
+            [DllImport(nameof(Kernel32))]
+            public static extern void SetLastError(int error);
```

Comment:
I thought this didn't work since the P/Invoke transition messes with it.

Reply:
Entirely predicated on the blittability of the signature. Right?

Reply:
We have fixed the P/Invoke transition to not mess with last error a long time ago.